platform/vs/docker-ptf-sai.mk

```
@@ -0,0 +1,9 @@
+# docker image for docker-ptf-sai
+
+DOCKER_PTF_SAI = docker-ptf-sai.gz
+DOCKER_PTF_BASE = docker-ptf.gz
+$(DOCKER_PTF_SAI)_PATH = $(DOCKERS_PATH)/docker-ptf-sai
+$(DOCKER_PTF_SAI)_DEPENDS += $(LIBTHRIFT_2) $(PYTHON3_THRIFT_2)
```

The "LIBTHRIFT_2" name were rejected on review and the real name is "LIBTHRIFT_0_13_0" or "LIBTHRIFT_0_14_1". So looks like there is a bug on the master branch too.

As in this PR https://github.com/Azure/sonic-buildimage/pull/10201, The Thrift version will be bumped to 0.14.1 on 2022111 branch, so I believe we need to use the 0.14.1 version in this PR

```suggestion
$(DOCKER_PTF_SAI)_DEPENDS += $(LIBTHRIFT_0_14_1) $(PYTHON3_THRIFT_0_14_1)
```

---

Thanks for the review, resolved.